wh00t_core/library/client_network.py

```diff
@@ -23,27 +27,38 @@ def __init__(self, host: str, port: int, app_id: str, app_profile: str, logging_
         self.app_id: str = app_id
         self.app_profile: str = app_profile
 
+    def log(self, log_type, message):
```

Comment:
Small thing on including type annotations here

```suggestion
    def log(self, log_type: str, message: str):
```